src/main/java/teammates/ui/controller/InstructorFeedbackEditPageAction.java

```diff
@@ -24,6 +24,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         Assumption.assertPostParamNotNull(Const.ParamsNames.COURSE_ID, courseId);
         String feedbackSessionName = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
         Assumption.assertPostParamNotNull(Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionName);
+        String loadInEditMode = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT);
+        Assumption.assertPostParamNotNull(Const.ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT, loadInEditMode);
```

Comment:
I feel that we should allow it to be `null` so that the `editsessiondetails` part of the URL parameters becomes optional (as it is now we have way too many parameters littering the URL - the usual practice for web apps should be to hide non-configurable parameters from the user such that they see a clean URL). If `"true".equals(loadInEditMode)`, set `loadInEditMode` to `true`, else `false` (whether it's `null` or other user-input value like `0`, `false` etc.).